### PR TITLE
add null check for task context.

### PR DESCRIPTION
**Pull request by nishantmonu51**

### CODE REVIEW

**Comment by himanshug** [comment]
LGTM

**Comment by guobingkun** [comment]
:+1:

File: indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java [hunk 1]
```diff
@@ -215,15 +215,18 @@ public TaskStatus call()
...
-                              for (String propName : task.getContext().keySet()) {
-                                if (propName.startsWith(CHILD_PROPERTY_PREFIX)) {
-                                  command.add(
-                                      String.format(
-                                          "-D%s=%s",
-                                          propName.substring(CHILD_PROPERTY_PREFIX.length()),
-                                          task.getContextValue(propName)
-                                      )
-                                  );
+                              Map<String, Object> context = task.getContext();
```
Comment (guobingkun) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java]: nitpick: This could be declared ```final```

Comment (nishantmonu51) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java]: modified to final.

**Pull request merged**